pkg/sentry/fs/fsutil/cache_reclaim.go

```diff
@@ -0,0 +1,350 @@
+package fsutil
+
+import (
+        "sync"
+        "sync/atomic"
+        "time"
+        "runtime"
+
+        "gvisor.dev/gvisor/pkg/ilist"
+        "gvisor.dev/gvisor/pkg/log"
+        "gvisor.dev/gvisor/pkg/sentry/context"
+        "gvisor.dev/gvisor/pkg/sentry/memmap"
+        "gvisor.dev/gvisor/pkg/sentry/pgalloc"
+        "gvisor.dev/gvisor/pkg/sentry/platform"
+        "gvisor.dev/gvisor/pkg/sentry/usermem"
+)
+
+const (
+        LRU_ACTIVE_LIST = 0
+        LRU_INACTIVE_LIST = 1
+        LRU_LIST_NUM = 2
+)
+
+type LruEntry struct {
+	ilist.Entry
+	fr platform.FileRange
+	mr memmap.MappableRange
+	c *CachingInodeOperations
+	accessed bool
+	lru int
+}
+
+type LruManager struct {
+	destroyed bool
+	forceReclaim bool
+
+	// LRULists store the current file ranges consumed by all the files
+        // with the minimum granularity of 4K
+        // LRUMap records the mappings from the start of file range to its entry
+        //
+        // LRULists and LRUMap are protected by LRUMapMu
```

Comment:
Nit: comments should generally be complete sentences (ending in a period). Here and elsewhere.

Reply:
Thanks for pointing it out.